enhancements/prevent-communitytasks-forbidden-events.md

```diff
@@ -0,0 +1,73 @@
+# Enhance community task fetch from external/public repo
+
+## Issue: [#SRVKP-1729](https://issues.redhat.com/browse/SRVKP-1729)
+
+If the firewall blocks requests to GitHub it will result in a lot of `forbidden` events logs
+
+
+## Proposal
+### Solution 1
+Add a toggle in the tektonconfig which a user can set to `false` if they don't want to install community tasks from repo
+
+- Implementation 1.1 
+    Add <span style="color: green"> enableCommunityTask: true </span> as spec element of addon
```

Comment:
from a developer prespective it makes sense. From the prespective of an enduser this could be frustrating. ie, by default this will be enabled, then the user will have to set this to `false` after getting the error.

However, this could be ok as the user will have to set this only once on a disconnected cluster as operator persists customizations on TektonAddon CR instances across upgrades.